<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find awsp_controller)/config/rosconsole.conf"/>

    <node
            pkg = "awsp_gnss_l86_interface"
            name = "awsp_gnss_l86_interface_node"
            type = "awsp_gnss_l86_interface_node"
            args = "/dev/serial0 9600 10"
            required = "true"
    />

    <node
            pkg = "awsp_gy_88_interface"
            name = "awsp_gy_88_interface_node"
            type = "awsp_gy_88_interface_node"
            args = "300"
            required = "true"
    />

    <node
            pkg = "awsp_obstacle_detector"
            name = "awsp_obstacle_detector"
            type = "awsp_obstacle_detector"
            args = "23 24 300 30000"
            required = "true"
    />
    
    <node
            pkg = "awsp_controller"
            name = "awsp_controller_node"
            type = "awsp_controller_node"
            output="screen"
            required = "true"
    />


</launch>